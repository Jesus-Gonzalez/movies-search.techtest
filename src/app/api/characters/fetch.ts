import axios from 'axios';

export const fetchCharacters = () => (
  axios({
    url: 'https://swapi.dev/api/people/',
    responseType: 'json'
  })
  .then(response => response.data)
)
