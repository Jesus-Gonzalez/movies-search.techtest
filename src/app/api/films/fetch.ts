import axios from 'axios';

export const fetchFilms = () => (
  axios({
    url: 'https://swapi.dev/api/films/',
    responseType: 'json'
  })
  .then(response => response.data)
)
