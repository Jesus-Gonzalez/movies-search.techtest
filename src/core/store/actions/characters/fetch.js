export const fetchCharacters = () => ({
  type: 'FETCH_CHARACTERS'
});
