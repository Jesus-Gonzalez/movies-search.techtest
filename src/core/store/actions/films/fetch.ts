import * as ActionTypes from 'core/store/actionTypes';

export const fetchFilms = () => ({
  type: ActionTypes.Films.Fetch
});
