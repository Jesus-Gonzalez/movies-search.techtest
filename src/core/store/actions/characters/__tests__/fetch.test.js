import { fetchCharacters } from '../fetch';

it('creates the action', () => {
  expect(fetchCharacters())
    .toMatchSnapshot();
});
