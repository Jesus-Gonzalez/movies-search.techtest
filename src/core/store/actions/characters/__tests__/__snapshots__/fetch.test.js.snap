// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates the action 1`] = `
Object {
  "type": "FETCH_CHARACTERS",
}
`;
